### ISSUE TITLE
There is a new release of ansible-conjur-collection

### ISSUE THREAD

**Issue by BradleyBoutcher** [cyberark/ansible-conjur-collection#35]
Pending the merge of ansible-conjur-host-identity into the collection, there is a new release for the ansible-conjur-collection.

- [ ] Add a tag in the collection that announces the role, and follows our release plan for this project

This issue is connected to: https://github.com/cyberark/ansible-conjur-host-identity/issues/30 and #30

**Comment by infamousjoeg** [comment]
Let me know if I can help somehow, @BradleyBoutcher.  My demo is pending inclusion of this role in the collection. :)

**Comment by izgeri** [comment]
The release depends on #45 and maybe #44 / #47 

Maybe your demo helps answer #47? Reading the README, the UX of using the lookup plugin and the role together looked odd (eg configure one with env vars and one in playbook - how do they play nicely together?)

**Comment by izgeri** [comment]
Closing as a duplicate of #53 - note that #44, #45, #57 are still outstanding, but since releasing this unblocks some other work and we have not changed the Ansible role functionality as we ported it, we will review those items in future releases that work to continue to improve this collection.
